### ISSUE TITLE
mobiscroll 阻挡键盘快捷键的正常行为

### ISSUE THREAD

**Issue by nightire** [choice-form/os-client#14]
mobiscroll 默认有绑定一些键盘快捷键，具体如下：

> - General (applicable to all popups): <kbd>Space</kbd> or <kbd>Enter</kbd> to open the popup, <kbd>Esc</kbd> to cancel, <kbd>Enter</kbd> to set, focus trap (focus is not allowed to leave the modal popup, elements outside the popup cannot be focused)
> - Scroller based components: <kbd>Tab</kbd>, <kbd>Shift + Tab</kbd> to navigate between wheels and buttons, <kbd>Up</kbd> and <kbd>Down</kbd> arrows to change wheel values, <kbd>Space</kbd> to activate buttons
> - Calendar: <kbd>Tab</kbd>, <kbd>Shift + Tab</kbd> to navigate between tabs, month and year change buttons, month days, <kbd>Space</kbd> to activate focused buttons, tabs, or select the focused day
> - Numpad: <kbd>Tab</kbd>, <kbd>Shift + Tab</kbd> to navigate between buttons, <kbd>Space</kbd> to press buttons
> - Menustrip: <kbd>Tab</kbd>, <kbd>Shift + Tab</kbd> to navigate between items, <kbd>Space</kbd> to activate the focused item
> - Forms: <kbd>Tab</kbd>, <kbd>Shift + Tab</kbd> to navigate between form elements, <kbd>Space</kbd> to press buttons, <kbd>Up</kbd>, <kbd>Down</kbd>, <kbd>Left</kbd>, <kbd>Right</kbd> arrows to move the focused slider
>
> https://docs.mobiscroll.com/3-2-6/javascript/accessibility#acc-keyboard

这其中大多数的设定都可以沿用，但主要是 <kbd>Enter</kbd> 会给人带来很多困扰，因为我们在问卷里会使用 <kbd>Enter</kbd> 来做题目之前的确认和进行到下一题的导航。通过和 mobiscroll 的开发者沟通，获知目前没有单独对具体键位绑定的设定可以调整，但是有一个方式可以废除掉默认的键位绑定然后自己来处理：

> A possible solution would be to switch of the showing of mobiscroll, using `showOnTap: false` and `showOnFocus: false`.
>
> After this you can control entirely when to show the mobiscroll popup, you can bind your own events on the input or on an external button and call the `show` method of mobiscroll.
